src/app/shared/components/template/components/layout/popup/popup.component.ts

```diff
@@ -53,4 +53,7 @@ export interface ITemplatePopupComponentProps extends ITemplateContainerProps {
   waitForDismiss?: boolean;
   /** Display fullscreen overlayed on top of all other app content */
   fullscreen?: boolean;
+  /** Pass text only to render that text in a popup, bypassing any templates */
+  textOnly?: boolean;
+  text?: string;
```

Comment:
nit(blocking): we're using 2 props to essentially handle one thing, displaying text within the popup (instead of a template).

I think probably better to just call it `popupText` (to distinguish from `template` which should eventually be renamed to `popupTemplate`). That way you also don't have the confusion if `textOnly:true` but no text defined or `text` passed but not `textOnly`